extensions/sql-assessment/src/assessmentResultGrid.ts

```diff
@@ -29,20 +29,35 @@ export class AssessmentResultGrid implements vscode.Disposable {
 
 	private asmtType!: AssessmentType;
 
-	private readonly checkIdColOrder = 4;
+	private readonly checkIdColOrder = 5;
 	private readonly targetColOrder = 0;
 
 	public get component(): azdata.Component {
 		return this.rootContainer;
 	}
 
-	public constructor(view: azdata.ModelView) {
+	public constructor(view: azdata.ModelView, extensionContext: vscode.ExtensionContext) {
 		const headerCssClass = 'no-borders align-with-header';
```

Comment:
seems the headerCssClass is not introduced by you, but as I pointed out earlier, extension shouldn't be aware of the css classes, to achieve the same effect, you can expose options for the table, or column level option, e.g. showBorder?:boolean. and then in the core, you can translate to a css class.

Reply:
I will create a separate PR to get rid of css classes in the extension. 
I suppose we would need to mark azdata.TableColumn.headerCssClass and azdata.TableColumn.cssClass properties as deprecated or something like that for other extensions authors and remove them eventually.